app/js/components/LabResultsList.jsx

```diff
@@ -225,6 +225,7 @@ export class LabResultsList extends PureComponent {
                     columnMetadata={rowColumnMetadata}
                     collapseOnDataChange={false}
                     collapseOnPageChange={false}
+                    defaultPageSize={row.original.groupMembers.length}
                     showPagination={false}
```

Comment:
The issue here was that the same SortableTable element was being used here that was being used for the main table, but with showPagination=false so that it doesn't show the pagination toolbar.  However, perhaps confusingly, even though showPagination disables the toolbar, it doesn't disable pagination, so the table was still being paginated, and since no default page size was specified, it used the standard default of 10.

A suggested fix was to set the default page size to the some of members of the obsgroup,